lib/common.js

```diff
@@ -445,8 +445,14 @@ exports.readMysqlValue = function(
       } else {
         if(column.charset !== null) {
           // Javascript UTF8 always allows up to 4 bytes per character
-          column.charset = column.charset === 'utf8mb4' ? 'utf8' : column.charset;
-          column.charset = column.charset === 'utf8mb3' ? 'utf8' : column.charset;
+          if (column.charset === 'utf8mb4' || column.charset === 'utf8mb3') {
+            column.charset = 'utf-8'
```

Comment:
Might not be important, but is the hyphen not affecting anything? `'utf-8'` vs `'utf8'`

Reply:
Woah, good catch @alanken10! This matters yes. I'll update